package.json

```diff
@@ -36,17 +36,22 @@
   "dependencies": {
     "@actions/core": "^1.11.1",
     "@clack/prompts": "^0.10.1",
+    "@node-core/rehype-shiki": "^1.0.1-1815fa769361b836fa52cfab9c5bd4991f571c95",
```

Comment:
Not importing any element from ui-components?

Reply:
> Or is this jsx generator just to generate new jsx nodes and then another generator to transform to html? (jsx layouts + all elements + what not)?

Bingo. Perhaps a rename to jsx-ast?

Reply:
This generator generates the nodes, and the react-web one takes those nodes, compiles it with the components using ESBuild, and outputs HTML

Reply:
ESBuild was mentioned in #7, but any compiler works. That's a discussion for that issue.